package/MDAnalysis/analysis/helanal.py

```diff
@@ -749,15 +749,14 @@ def rotation_angle(helix_vector, axis_vector, rotation_vector):
 def vector_of_best_fit(origins):
     origins = np.asarray(origins)
     centroids = center(origins)
-    M = np.matrix(origins - centroids)
-    A = M.transpose() * M
+    M = np.array(origins - centroids)
```

Comment:
same here, probably don't need the `np.array` call if the subtraction already works

Reply:
done